### PR TITLE
Added action to run or schedule job

### PR DESCRIPTION
**Pull request by johnsca**

### CODE REVIEW

**Comment by johnsca** [comment]
We might want to add support for one or more of these args to SPARK_HOME and / or HADOOP_HOME expanded.  We probably also need to include the HADOOP_EXTRA_CLASSPATH.

File: actions/submit [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+job="$(action-get job)"
+job_args="$(action-get job-args)"
+cron="$(action-get cron)"
+
+submit_args='--master ${MASTER} --deploy-mode cluster'
+if [[ -n "$packages" ]]; then
+    submit_args="$submit_args --packages $packages"
+fi
+if [[ -n "$py_files" ]]; then
+    submit_args="$submit_args --packages $py_files"
```
Comment (ktsakalozos) [inline actions/submit]: should this be "--py-files"?

File: actions/submit [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+fi
+if [[ -n "$extra_params" ]]; then
+    submit_args="$submit_args $extra_params"
+fi
+if [[ -n "$class" ]]; then
+    submit_args="$submit_args --class $class"
+fi
+submit_args="$submit_args $job"
+
+job_code=". /etc/environment ; spark-submit ${submit_args} ${job_args}"
```
Comment (ktsakalozos) [inline actions/submit]: Here is a crazy idea. If we had "spark-submit" as a parameter that defaults to "spark-submit" we could do more interesting things. For example, a user could upload to HDFS his script and then execute it using this action. Far more powerful, I think :)

File: actions/submit (hunk 1 above)
Comment (johnsca) [inline actions/submit]: Yes, thanks.  Copypasta error.

**Pull request merged**